src/ether-utils.js

```diff
@@ -0,0 +1,6 @@
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
```

Comment:
This is a regular function, not a component, so by convention it should start with a lowecase letter.